composer.json

```diff
@@ -34,7 +34,9 @@
         "fix-cs": "@php ./vendor/bin/php-cs-fixer fix -v --show-progress=estimating"
     },
     "extra": {
+        "_ezplatform_branch_for_behat_tests": "ezp-28009-enable-richtext-bundle",
```

Comment:
Open question: should we wait with merging this until ezsystems/ezplatform#337 is merged or should we merge it right now and update it later?